scripts/opencv_xcframework.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/zsh
+
+set -euo pipefail
+
+readonly BUILD_DIR="opencv_build"
+readonly OUTPUT_DIR="${BUILD_DIR}/output"
+
+clone() {
+    readonly VERSION="${1}"
+    git clone --branch "${VERSION}" --depth 1 https://github.com/opencv/opencv.git "${BUILD_DIR}/${VERSION}"
+}
+
+build_xcframework() {
+    readonly VERSION="${1}"
+    shift
+
+    python3 "${BUILD_DIR}/${VERSION}/platforms/apple/build_xcframework.py" \
+        --build_only_specified_archs \
+        --iphoneos_archs arm64,armv7 \
+        --iphonesimulator_archs arm64,x86_64 \
+        --out "${OUTPUT_DIR}" \
+        "$@"
+}
+
+# OpenCV builds frameworks with symlinks inside them: remove them here, for clarity
+patch_xcframework_remove_symlinks() {
+    readonly TMP_DIR="${OUTPUT_DIR}/tmp_dir"
+    find "$(pwd)/${OUTPUT_DIR}/opencv2.xcframework" -name '*.framework' -print0 | while read -d $'\0' framework; do
+        rm -rf "${TMP_DIR}"
+        mv "${framework}" "${TMP_DIR}"
+        rsync -r "${TMP_DIR}/Versions/A/" "${framework}"
```

Comment:
Is there a reason to use `rsync` instead of `mv`?

Reply:
The alternative for this line using `mv` is:

```
        mkdir -p "${framework}"
        mv -v ${TMP_DIR}/Versions/A/* "${framework}"
```
Note how NOT adding `"` in the parameter with the `*` is mandatory to trigger shell expansion. I think `rsync` is safer/clearer

Reply:
Sorry, looked at it again. Nope, no reason for `rsync`, changed to `mv`